data-exporter/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>org.reactome.release</groupId>
 	<artifactId>data-exporter</artifactId>
 	<packaging>jar</packaging>
-	<version>1.0-SNAPSHOT</version>
+	<version>1.1.0-SNAPSHOT</version>
```

Comment:
We should probably start moving away from the string `SNAPSHOT` being in our versioning, all my new repos have it as well

Reply:
Yeah, I'll remove it seeing is this is a finished minor update.  I had to refresh my memory on when to use SNAPSHOT and when not to use it.

https://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it